### ISSUE TITLE
Add liblas formula

### ISSUE THREAD

**Issue by nextstopsun** [OSGeo/homebrew-osgeo4mac#298]
Now you can't have a straightforward build of grass7 with lidar support because it's dependency `liblas` depends on ancient `gdal` formula. Can `liblas` be moved here and promoted to a `gdal2` dependency?

**Comment by dakcarto** [comment]
Should be fixed in 2285e2f and fb42eab. Give it a try.

**Comment by nextstopsun** [comment]
```brew install --with-laszip liblas-gdal2```
doesn't work for me. 
```
      FILE failed to open for reading (No such file or directory):

    /usr/local/include/laszip/laszip.hpp
```

**Comment by dakcarto** [comment]
@nextstopsun should be fixed in 02293d9.

**Comment by nextstopsun** [comment]
@dakcarto Got it! Thanks!
